microk8s-resources/actions/enable.istio.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Enabling Istio"
+
+# pod/servicegraph will start failing without dns
+"$SNAP/microk8s-enable.wrapper" dns
+
+read -p "Enforce mutual TLS authentication between sidecars? (Y/N): " confirm
```

Comment:
Maybe we should just assume Y and not prompt for this (i.e., always use mutual TLS). It would be simpler and not require user interaction. Any downside?

Reply:
Okay. Well "N" is the safer option if the user doesn't know the difference, so that should be the default. It would also be nice to say something like "(if unsure, choose N)", and maybe even provide a link (somewhere) to the options where they can read about the differences.